local-dev/local-postgres-init.sql

```diff
@@ -2,3 +2,6 @@ CREATE DATABASE testdb;
 CREATE ROLE dbuser WITH LOGIN ENCRYPTED PASSWORD 'dbpwd';
 CREATE DATABASE tclstairway;
 CREATE ROLE tclstairwayuser WITH LOGIN ENCRYPTED PASSWORD 'tclstairwaypwd';
+GRANT ALL ON DATABASE tclstairway TO tclstairwayuser WITH GRANT OPTION;
```

Comment:
what is this for?

Reply:
I used the script to create a new database. The tests failed because the tclstairwayuser was not given access to the database or to the public schema.